src/RosClientNode.hpp

```diff
@@ -201,12 +240,57 @@ class RosClientNode : public QObject {
 
     // Set up publishers for these remote messages
     if (pubs.count(full_from_topic) == 0) {
-      pubs[full_from_topic] = n.advertise<T>(full_to_topic, 1); 
+      pubs[full_from_topic] = n.advertise<T>(full_to_topic, 1);
     }
 
     pub_remote_topics.push_back(full_from_topic);
   }
 
+  /**
+   * @deprecated Use configure() instead. See config.example.hpp
```

Comment:
Tbh I'd personally be a fan of not keeping around the deprecated support and just making it the job of whoever "pulls" the latest code to update the configs.
We don't have *that* many robots using robofleet, and keeping around deprecated code I think really only makes sense if we have "not us" users who we want to keep supporting

Reply:
I agree, removed it.